tests/genkey_sign_ver/genkey-sign-ver-test.sh

```diff
@@ -96,12 +96,12 @@ gen_key_sign_ver_test(){
     [ $RESULT -ne 0 ] && printf '%s\n' "Failed $1 sign" && exit -1
 
     if [ $1 = "rsa" ]; then
-        ORIGINAL=`cat sign-this.txt`
+        ORIGINAL=`cat -A sign-this.txt`
```

Comment:
Reverting the change in clu_verify.c this test still passes for me `PASS: tests/genkey_sign_ver/genkey-sign-ver-test.sh` , we should have a test that would fail if the current behavior in main is not fixed.

Maybe check out 'ls -s file.txt' and grep the output for the size to compare.

Reply:
Hmm did you run `make` after reverting the change? Failing locally for me, as well as on github actions.